src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -117,6 +118,7 @@
 
 	private final PlayerSpigotMock playerSpigotMock = new PlayerSpigotMock();
 	private final List<AudioExperience> heardSounds = new LinkedList<>();
+	private final HashMap<UUID,List<String>> hiddenPlayerNicks = new HashMap<>();
```

Comment:
Collections, like Maps or Lists should always be declared with their base type to abstract away from the concrete implementation (makes it easier to change later to a different type of Map in case we need to.

Also the method name is a bit misleading, with `hiddenPlayerNicks` I would expect it to store nicknames of hidden players? But it just stores the UUIDs of hidden Players with a set of plugin names. So we should probably simplify that name.

```suggestion
	private final Map<UUID, List<String>> hiddenPlayers = new HashMap<>();
```